lib/ts/recipe/emailpassword/translations.ts

```diff
@@ -0,0 +1,45 @@
+import { defaultTranslationsCommon } from "../../components/translations";
```

Comment:
Maybe consider moving all the translations into the theme folder of each recipe.